package.json

```diff
@@ -61,5 +61,12 @@
   },
   "peerDependencies": {
     "three": ">=0.123.0"
+  },
+  "optionalDependencies": {
+    "@react-three/drei": "^9.114.3",
```

Comment:
Drei is only used in the examples and isn't used in any of the published components so this can be moved to "devDependencies", instead.